srcpkgs/sublime-merge-bin/template

```diff
@@ -0,0 +1,41 @@
+# Template file for 'sublime-merge-bin'
+pkgname=sublime-merge-bin
+version=2032
+revision=1
+archs="x86_64"
+wrksrc=sublime_merge
+hostmakedepends="w3m"
+depends="libpng gtk+ hicolor-icon-theme desktop-file-utils"
+short_desc="New Git client, from the makers of Sublime Text"
+maintainer="Adelmo Junior <noblehelm@gmail.com>"
+license="custom:EULA"
+homepage="https://www.sublimemerge.com"
+distfiles="https://download.sublimetext.com/sublime_merge_build_${version}_x64.tar.xz"
+checksum=b782c768383893ba7803c2cffd428b09bec46be8a65e61a5f17964bdcc2aaf7c
+_license_checksum=aaf2f4a457945145bfea1ff06498806bd91951f76ec3cefec169f904ff4cb158
+nopie=yes
+repository=nonfree
+restricted=yes
+replaces="sublime-merge>0"
+
+post_extract() {
+	$XBPS_FETCH_CMD https://www.sublimehq.com/eula
+	w3m -dump -I utf-8 -T text/html < eula > EULA
+
+	filesum="$(xbps-digest EULA)"
```

Comment:
$XBPS_DIGEST_CMD < EULA

Reply:
This validation is incredibly fragile because the checksum is almost certain to change whenever Sublime changes something trivial like the navigation header. In a restricted package, we don't care about the contents of the license because we aren't distributing anything. There's no need to do any validation here.

I recommend just dropping `_license_checksum` and everything in this function except the `w3m` dump.

Reply:
Works since some time. Some packages sed out header and footer to increase chances of change.